src/main/java/fr/cirad/web/controller/gigwa/GigwaRestController.java

```diff
@@ -1807,7 +1802,7 @@ public void run() {
 						try {							
 							Integer newProjId = null;
 							if (fBrapiImport)
-								newProjId = new BrapiImport(token).importToMongo(sNormalizedModule, sProject, sRun, sTechnology == null ? "" : sTechnology, dataUri1.trim(), sBrapiStudyDbId, sBrapiMapDbId, sBrapiToken,  Boolean.TRUE.equals(fClearProjectData) ? 1 : 0);
+								newProjId = new BrapiImport(processId).importToMongo(sNormalizedModule, sProject, sRun, sTechnology == null ? "" : sTechnology, dataUri1.trim(), sBrapiStudyDbId, sBrapiMapDbId, sBrapiToken,  Boolean.TRUE.equals(fClearProjectData) ? 1 : 0);

```

Comment:
Is this process id change going to be maintained separately from upstream code? Any plans to see if some sort of configurable option could be implemented?

Reply:
@GuilhemSempere updated Gigwa to do this in the upstream repo.  I've rebased to this, so now it is in-line.